pkg/cloudprovider/cloudprovider.go

```diff
@@ -259,7 +259,6 @@ func (c *CloudProvider) Get(ctx context.Context, providerID string) (*karpv1.Nod
 	if err != nil {
 		return nil, fmt.Errorf("getting vm name, %w", err)
 	}
-	ctx = log.IntoContext(ctx, log.FromContext(ctx).WithValues("id", vmName))
```

Comment:
This one needs to stay, stashes new logger into context. Also need to update "id" with "vmName" for consistency